.github/workflows/checkstyle.yaml

```diff
@@ -13,32 +13,44 @@ jobs:
         ref: ${{ github.event.pull_request.head.sha }}
     - name: Install dependencies
       run: |
+        echo 'force-unsafe-io' | sudo tee /etc/dpkg/dpkg.cfg.d/force_unsafe_io
         sudo apt-get update
-        sudo apt-get install --yes -qq build-essential autoconf libtool gawk alien fakeroot linux-headers-$(uname -r)
-        sudo apt-get install --yes -qq zlib1g-dev uuid-dev libattr1-dev libblkid-dev libselinux-dev libudev-dev libssl-dev python3 python3-dev python3-setuptools python3-cffi
-        # packages for tests
-        sudo apt-get install --yes -qq parted lsscsi ksh attr acl nfs-kernel-server fio
-        sudo apt-get install --yes -qq mandoc cppcheck pax-utils devscripts
-        sudo -E pip --quiet install flake8
+        xargs --arg-file=./.github/workflows/build-dependencies.txt sudo apt-get install -qq
```

Comment:
I prefer to leave `--yes` just to be sure.

Reply:
`apt-get(8)`
```
       -q, --quiet
           Quiet; produces output suitable for logging, omitting progress indicators. More q's will produce more quiet up to a maximum of 2. You can also use -q=# to set the quiet
           level, overriding the configuration file. Note that quiet level 2 implies -y; you should never use -qq without a no-action modifier such as -d, --print-uris or -s as APT may
           decide to do something you did not expect. Configuration Item: quiet.
```

Reply:
Oh, great, thank you for info!